ansible/roles/nats/templates/0nats-server.conf

```diff
@@ -0,0 +1,26 @@
+listen: 0.0.0.0:4222 
+http: 0.0.0.0:8222 
+
+debug: false
+log_file: "/opt/nats/logs/nats-server.log"
+
+pid_file: "/opt/nats/nats-server.pid"
+
+cluster {
+    listen: 0.0.0.0:6222 
+    
+    routes = [
+      nats-route://{{ nats_host2 }}:6222
+      nats-route://{{ nats_host3 }}:6222
+    ]
+}
+
+streaming {
+  id: mqperf-cluster
```

Comment:
hm I'm reading over the configuration, shouldn't this be `cluster_id`?  See https://docs.nats.io/nats-streaming-server/configuring/cfgfile - but maybe it works like this as well